conf.d/python.d/neo.conf

```diff
@@ -0,0 +1,9 @@
+neo:
+    name : 'neo'
+    host : 'localhost'
+    port : '7687' # port where Bolt protocol is listening
+    user : 'neo4j'
+    pwd : 'password'
+    update_every : 30
+    priority: 22000 # the JOB's order on the dashboard
```

Comment:
Not sure if it's an issue, but this is set different from what you have in the module itself.  Also, if it's set there, you don't have to have it set here.

Reply:
so I will remove updat_every and priority from the code. Thanks for the spotting. Did I say already I learned the little Python I know with this plugin's plugin :)